src/Compilers/Core/Portable/CodeGen/ILBuilderEmit.cs

```diff
@@ -38,13 +38,13 @@ internal void EmitOpCode(ILOpCode code, int stackAdjustment)
 
         internal void EmitToken(string value)
         {
-            uint token = module == null ? 0xFFFF : module.GetFakeStringTokenForIL(value);
+            uint token = module?.GetFakeStringTokenForIL(value) ?? 0xFFFF;
             this.GetCurrentStream().WriteUint(token);
         }
 
         internal void EmitToken(Microsoft.Cci.IReference value, SyntaxNode syntaxNode, DiagnosticBag diagnostics)
         {
-            uint token = module == null ? 0xFFFF : module.GetFakeSymbolTokenForIL(value, syntaxNode, diagnostics);
+            uint token = module?.GetFakeSymbolTokenForIL(value, syntaxNode, diagnostics) ?? 0xFFFF;
```

Comment:
In this case yes I think it is more readable after a couple of viewings.  Other cases though still seem less readable.  

Don't want to get too hung up on this issue.  My comment was meant as a non-blocking note on the PR